fitbenchmarking/utils/options.py

```diff
@@ -290,10 +290,6 @@ def results_dir(self):
         """
         return self._results_dir
 
-    @results_dir.setter
```

Comment:
I think this is here just to enforce the `os.path.abspath`. Is this unneccessary now?

If we are removing the setter, we could also remove the getter and access the `results_dir` directly rather than through a private variable.